### ISSUE TITLE
"Always show input" causes mic icon to show up all the time

### ISSUE THREAD

**Issue by xenotropic** [qwreey75/quick-settings-tweaks#76]
In the previous version of Gnome (Ubuntu Jammy, so I guess Gnome 42), the behavior was: (1) the mic setting is always in the quick settings menu; and (2) only when an input stream was active (some device is "listening") the mic icon shows up in the tray (see pic to clarify what icon I refer to). 

![image](https://user-images.githubusercontent.com/4050617/229890633-35c1fda0-2a43-4d26-af3b-ebbb36a2da36.png)

This was a great set of behaviors. 

In Gnome 43, the mic setting is in the quick settings panel when an input stream is active. This breaks (1), making it harder to change the input device easily. You have to actually start recording to change microphones, or go to the full audio settings panel. 

So I installed this extension. It has an "always show input" switch. That puts back behavior (1). Yay! However, turning that switch on also breaks behavior (2). It causes the mic icon to always show up, as if there was a recording stream active all the time. This makes it impossible to tell when an input stream actually is active; the user no longer knows when an application on the computer is "listening" to them or not. 

Can you possibly make the extension do both (1) and (2)?

**Comment by Rayzeq** [comment]
[Quick Settings Audio Panel](https://extensions.gnome.org/extension/5940/quick-settings-audio-panel/) (of which I'm the developer) can do that, **however** there are some incompatibilities with Quick Settings Tweaks that you can find on the [github page](https://github.com/Rayzeq/quick-settings-audio-panel) of the extension.
Do note that you don't need to enable any other feature of the extension to enable this one.

**Comment by xenotropic** [comment]
Thanks -- I'm still on Gnome 43 so appears incompatible, but once I go up to 44 I'll check it out

**Comment by Rayzeq** [comment]
Yeah I think it should work on gnome 43 but since I can't test it I didn't add it to the list of compatible versions.
